pkgs/tools/text/patchutils/generic.nix

```diff
@@ -11,6 +11,8 @@ stdenv.mkDerivation rec {
     inherit sha256;
   };
 
+  outputs = [ "man" "out" ];
```

Comment:
The first argument from output list is the default one.

Yay! Obsessed Formatting Nitpickers are useful sometimes!